src/SFML/CMakeLists.txt

```diff
@@ -21,10 +21,14 @@ if (SFML_OS_WINDOWS)
                 set(CMAKE_LIBRARY_PATH ${CMAKE_LIBRARY_PATH} "${PROJECT_SOURCE_DIR}/extlibs/libs-msvc/x64")
             endif()
         else()
-            if(ARCH_32BITS)
-                set(CMAKE_LIBRARY_PATH ${CMAKE_LIBRARY_PATH} "${PROJECT_SOURCE_DIR}/extlibs/libs-msvc-universal/x86")
-            elseif(ARCH_64BITS)
-                set(CMAKE_LIBRARY_PATH ${CMAKE_LIBRARY_PATH} "${PROJECT_SOURCE_DIR}/extlibs/libs-msvc-universal/x64")
+            if(ARCH_ARM64)
+                set(CMAKE_LIBRARY_PATH ${CMAKE_LIBRARY_PATH} "${PROJECT_SOURCE_DIR}/extlibs/libs-msvc-universal/ARM64")
```

Comment:
As a minor point of style, is it `ARM64` or `Arm64`? What is the right capitalization to use?